tests/by-util/test_install.rs

```diff
@@ -1,4 +1,6 @@
 use crate::common::util::*;
+use filetime::FileTime;
+use nix::sys::stat::{mknod, Mode, SFlag};
```

Comment:
please fix these warnings

Reply:
Those belong to `test_install_copy_then_compare_nonregular_file`, with which I need some help (see L426).